web/angular-wallet/src/app/ledger/ledger.service.desktop.ts

```diff
@@ -0,0 +1,26 @@
+import {Injectable} from '@angular/core';
```

Comment:
the trick (that I think _should_ work) is to have separate files for desktop and web.